backend/mem/file.go

```diff
@@ -0,0 +1,219 @@
+package mem
+
+import (
+	"bytes"
+	"errors"
+	"github.com/c2fo/vfs/v4"
+	"io"
+	"time"
+)
+
+//File implements vfs.File interface for os fs.
+type File struct {
+	exists 		bool
+	timeStamp 	time.Time
+	isOpen		bool
+	isZB		bool
+	isRef		bool
+	privSlice 	[] byte
+	byteBuf  *bytes.Buffer
+	Filename string
+	cursor 		int
+	location vfs.Location
+}
+
+func DoesNotExist() error {
```

Comment:
Is there a reason for this and the following two functions to be public? I'm not against it, I just don't see the use case offhand. If there is a use case, this should be described in the function comments